shared/package.json

```diff
@@ -9,11 +9,18 @@
   "imports": {
     "#*": "./src/*"
   },
+  "exports": {
+    "./build/": {
+      "import": "./build/esm/",
+      "require": "./build/cjs/",
+      "default": "./build/esm/"
+    }
+  },
   "scripts": {
     "lint:type": "npx tsc --noEmit",
     "lint:js": "npx eslint \"src/**/*.ts\"",
-    "build": "tsc && npm run build:copy",
-    "build:copy": "cp -r package.json build"
+    "build": "tsc -b ./tsconfig.cjs.json ./tsconfig.esm.json",
+    "build:copy": "cp -r package.json build/esm && cp -r package.json build/cjs"
```

Comment:
Checked locally, instead of build and build:copy it would be:
```
    "build:ts": "tsc -b ./tsconfig.cjs.json ./tsconfig.esm.json",
    "copy:packagejson:cjs": "cp package.json build/cjs && perl -i -pe's/\"module\",/\"commonjs\",/g' build/cjs/package.json",
    "copy:packagejson:esm": "cp package.json build/esm",
    "copy:packagejson": "npm run copy:packagejson:cjs && npm run copy:packagejson:esm",
    "build": "npm run build:ts && npm run copy:packagejson"
```

Reply:
For now package.json doesn't copy to folders

Reply:
There's the issue with copying package.json to folders, metro complains and fails

<img width="488" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/61088675/3c0f78c4-f9dd-4ecd-a9c9-a4ff6611898f">